Eatery/Onboarding/Controllers/OnboardingPageViewController.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OnboardingPageViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/12/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+import CHIPageControl
+
+class OnboardingPageViewController: UIPageViewController {
+
+    var pages = [OnboardingViewController]()
+    let pageControl = CHIPageControlJaloro()
+
+    let models = [
+        OnboardingModel(title: "Menus", subtitle: "See what’s being served at any campus eatery.", image: UIImage(named: "menuIcon")!),
+        OnboardingModel(title: "Collegetown", subtitle: "Find info about your favorite Collegetown spots.", image: UIImage(named: "ctownIcon")!),
+        OnboardingModel(title: "Transactions", subtitle: "Track your swipes, BRBs, meal history, and more.", image: UIImage(named: "transactionsIcon")!),
+        OnboardingModel(title: "Login", subtitle: "To get the most out of Eatery, log in with your NetID.", image: nil)
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.view.backgroundColor = .eateryBlue
+
+        pages = [
+            OnboardingViewController(model: models[0], nibName: nil, bundle: nil),
+            OnboardingViewController(model: models[1], nibName: nil, bundle: nil),
+            OnboardingViewController(model: models[2], nibName: nil, bundle: nil),
+            OnboardingLoginViewController(model: models[3], nibName: nil, bundle: nil)
+        ]
+
+        for page in pages { page.delegate = self }
```

Comment:
can we change this to `pages.forEach({ $0.delegate = self })`

Reply:
I thought about that to, but is it more readable or meaningfully shorter than the standard for loop?